packages/feedback/rollup.bundle.config.mjs

```diff
@@ -1,21 +1,8 @@
 import { makeBaseBundleConfig, makeBundleConfigVariants } from '@sentry-internal/rollup-utils';
 
 export default [
-  ...makeBundleConfigVariants(
-    makeBaseBundleConfig({
-      bundleType: 'addon',
-      entrypoints: ['src/index.ts'],
-      jsVersion: 'es6',
-      licenseTitle: '@sentry-internal/feedback',
-      outputFileBase: () => 'bundles/feedback',
-      sucrase: {
```

Comment:
These don't appear to be necessary (anymore?), it seems to be working fine without this as well (we also don't set this in the existing index.bundle.feedback config in browser)